atomic_reactor/schemas/content_sets.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+
+  "title": "Content sets container definition",
+
+  "type": ["object", "null"],
+  "patternProperties": {
+    "^[a-z].*$": {
```

Comment:
Do we want a more strict pattern here? like `[a-z][a-z0-9_]*`?

Reply:
If we are not enumerating the available platforms, I don't see a reason to be more strict here